documentation/quickstart-qemu.md

```diff
@@ -0,0 +1,102 @@
+# Getting Started with Qemu Worker
+
+This is a quick start guide for using the leviathan framework with the qemu worker.
+
+## Start your first test run
+
+For your first test run, we will be running the [e2e test suite](https://github.com/balena-os/leviathan/tree/master/suites/e2e).
+
+A balenaOS image that is used to flash and provision the device under test will be needed for the test. You can download an unmanaged image from [balena.io/os](balena.io/os). Extract the downloaded balenaOS image you want to test with to the `workspace` directory and rename it to `balena.img` file.
```

Comment:
Suggestion: for this very first example just use `image: false` and let the test suite download the image. We can explain about downloading OS images and etc on the "real tests" example below.